packages/roosterjs-editor-api/lib/utils/splitList.ts

```diff
@@ -0,0 +1,22 @@
+import blockFormat from '../utils/blockFormat';
+import { createVListFromRegion } from 'roosterjs-editor-dom';
+import { IEditor } from 'roosterjs-editor-types';
+
+/**
+ * @internal
+ */
+export default function splitList(
+    editor: IEditor,
+    list: HTMLOListElement,
+    startNumber?: number
+): void;
+
+export default function splitList(editor: IEditor, list: HTMLOListElement, startNumber?: number) {
```

Comment:
this parameter "list" is not used at all.

Reply:
You can declare startNumber default to be 1:

function splitlist(..., startNumber: number = 1) {